.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "ignore": ["node-modules/**", "src/index.umd.js"],
+  "presets": ["es2015", "stage-0", "react"],
```

Comment:
Instead of `stage-0` and `es2015` could we use `env`? If we don't provide any configuration, it will use the latest browsers. It's also the recommend preset.

Reply:
@agirton yup, we've made a note of using the env preset, please be on look out for a PR to implement this once this one has been closed out 👍